src/arrays.ts

```diff
@@ -0,0 +1,28 @@
+export type validArrayValue<T> = T[] | null | undefined;
+
+export default function shallowEqualArrays<T>(
+  arrA: validArrayValue<T>,
+  arrB: validArrayValue<T>
```

Comment:
The type doesn't feel right to me since users could provide array items of different types. Doesn't TypeScript complain if you do e.g. `shallowEqualArrays([1, "a"], [1, false])`?

Reply:
Ack. You're absolutely right. #33 to fix it.